src/Spectre.Data.Tests/Datasets/BasicTextDatasetTest.cs

```diff
@@ -0,0 +1,292 @@
+﻿/*
+ * BasicTextDatasetTest.cs
+ * Checks, whether MCR is properly called and result may be obtained.
+ * 
+   Copyright 2017 Dariusz Kuchta, Michał Gallus
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using NUnit.Framework;
+using Spectre.Data.Datasets;
+using Spectre.Data.Structures;
+
+namespace Spectre.Data.Tests.Datasets
+{
+    [TestFixture, Category("Dataset")]
+    public class BasicTextDatasetTest
+    {
+        private IDataset _dataset;
+
+        [OneTimeSetUp]
+        public void SetUpFixture()
+        {
+            Directory.SetCurrentDirectory(TestContext.CurrentContext.TestDirectory + "\\..\\..");
+        }
+
+        [SetUp]
+        public void SetUpClass()
+        {
+            double[] mz = {1.0, 2.0, 3.0};
+            double[,] data = {{1, 2.1, 3.2}, {4, 5.1, 6.2}, {7, 8.1, 9.2}};
+
+            _dataset = new BasicTextDataset(mz, data);
+        }
+
+        [Test]
+        public void CreateFromFileTest()
+        {
+            try
```

Comment:
Is it better to use attributes or try catch in tests? I think attributes are more clear.
http://stackoverflow.com/questions/26097930/how-to-handle-thrown-exception-in-nunit